korean-air/src/components/flightInfoPage/FlightCard.tsx

```diff
@@ -0,0 +1,302 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import theme from "../../styles/theme";
+import {
+  ChevronRight,
+  IcChangeFlight,
+  IcConditionArrow,
+  IcRefundFlight,
+  IcShortInfoFlight,
+  LogoJinair,
+} from "../../assets";
+
+export interface FlightCardProp {
+  click: boolean;
+  setClick: React.Dispatch<React.SetStateAction<boolean>>;
+  handleBtnClick: () => void;
+}
+
+const FlightCard: React.FC<FlightCardProp> = ({
+  click,
+  setClick,
+  handleBtnClick,
+}) => {
+  return (
+    <>
+      <Wrapper>
+        <article>
+          <div>
+            <TimeLayout>
+              <FlightTime>06:05</FlightTime>
+              <Area>SEL</Area>
+            </TimeLayout>
+            <ExpectedTimeLayout>
+              <Time>1시간 10분</Time>
+              <IcConditionArrow />
+            </ExpectedTimeLayout>
+            <TimeLayout>
+              <FlightTime>07:15</FlightTime>
+              <Area>CJU</Area>
+            </TimeLayout>
+            <ChevronRight />
+          </div>
+          <section>
+            <JinairInfo>
+              KE5153 <LogoJinair /> 진에어 운항
+            </JinairInfo>
+          </section>
+          <div>
+            <BtnGroup>
+              <BtnLayout
+                className={`${click ? "click" : ""}`}
+                onClick={handleBtnClick}
+              >
+                <CommonTitle className={`${click ? "click" : ""}`}>
+                  특가운임
+                </CommonTitle>
+                <CommonPrice className={`${click ? "click" : ""}`}>
+                  48,300원
+                </CommonPrice>
+                <CommonSeat className={`${click ? "click" : ""}`}>
+                  7석
+                </CommonSeat>
+              </BtnLayout>
```

Comment:
```suggestion
              <BtnLayout
                click={click}
                onClick={handleBtnClick}
              >
                <CommonTitle click={click}>
                  특가운임
                </CommonTitle>
                <CommonPrice click={click}>
                  48,300원
                </CommonPrice>
                <CommonSeat click={click}>
                  7석
                </CommonSeat>
              </BtnLayout>
```

오빠는 지금 click 값에 따라 스타일을 적용해주기 위해서 class를 사용해주고 있는데! 
우리가 3차세미나때 styled components 배우면서 동적 스타일링에 대해 배웠었는데 기억할까?
그때 styled component에게도 props를 넘겨줘서 그 값에 따라 스타일링하는 방법을 배웠었어!! 
[가물가물하면 다시 한번 공부해보자 이건 앞으로도 많이 쓸 내용이기 때문에! ](https://www.notion.so/dosopt/Styled-Components-bccd4244927548cabb4152177fb5134b?pvs=4#94767028b812496ba20985641472d21d)

이렇게 우리는 styled components를 사용해줌으로써 class를 사용하지 않고도 위에 내가 제안한 코드처럼 props를 바로 넘겨줄 수 있어. 받아온 컴포넌트를 어떻게 활용하면 좋을지는 아래부분에서 다시 코멘트 달게!

Reply:
아 링크 보니까 기억났어 ㅋㅋㅋㅋㅋㅋ 이따 적용시키겠습니다